templates/boarding-pass/pdf_boarding_pass.go

```diff
@@ -0,0 +1,171 @@
+/*
+ * This example showcases the usage of creator templates by creating a sample
+ * boarding pass document.
+ *
+ * Run as: go run pdf_boarding_pass.go
+ */
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"log"
+	"os"
+	"text/template"
+
+	"github.com/boombuler/barcode"
+	"github.com/boombuler/barcode/qr"
+	"github.com/unidoc/unipdf/v3/common"
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/core"
+	"github.com/unidoc/unipdf/v3/creator"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io.
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+
+	common.SetLogger(common.NewConsoleLogger(common.LogLevelDebug))
+}
+
+func main() {
+	c := creator.New()
+	c.SetPageMargins(30, 30, 20, 20)
+
+	// Read main content template.
+	mainTpl, err := readTemplate("templates/main.tpl")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Read boarding pass data json.
+	pass, err := readBoardingPassData("boarding_pass.json")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Create QR code image.
+	qrCode, err := createQRCode("https://github.com/unidoc/unipdf-examples/tree/master/templates/boarding-pass", 100, 100)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Draw main content teplate.
```

Comment:
Minor typo in the comment: `teplate` instead of `template`. I think this is a typo in other examples as well.
Could you fix those as well please?